src/common/helpers/utils.ts

```diff
@@ -53,38 +55,38 @@ export function convertIpfsLink(link: string): string {
   let metadata = link;
 
   if (metadata.startsWith("ipfs://ipfs/")) {
-    metadata = "https://flk-ipfs.xyz/ipfs" + metadata.slice(11);
+    metadata = "https://ipfs1.fiit.stuba.sk/ipfs" + metadata.slice(11);

```

Comment:
maybe you could create a constant from this so that it is more easily replaceable in the future